python/test/test_polars.py

```diff
@@ -333,15 +333,15 @@ def test_polars_context():
         context.query()
         .with_columns(pl.col("B").is_null().alias("B_nulls"))
         .filter(pl.col("B_nulls"))
-        .select(pl.col("A").fill_null(2.0).dp.sum((0, 3)))
+        .select(pl.col("A").dp.sum((0, 3)))
```

Comment:
If someone wanted to specify a default for missing values, is that still possible?

Reply:
Yes, if they have their own fill_null, then it will fill before .dp.sum's fill_null. Then the fill_null in .dp.sum becomes a no-op.